actions/npm-dependency-vulnerabilities/package.json

```diff
@@ -7,13 +7,13 @@
     "all": "npm run lint && npm run prepare && npm run test"
   },
   "dependencies": {
-    "@actions/core": "1.6.0",
+    "@actions/core": "^1.10.1",
```

Comment:
Let's pin the version to ensure the stability

Reply:
By the way, it's not crucial for dev deps, but the ones we ship to clients should be as stable as possible.